### DISCUSSION TITLE
Why does pytestmark with usefixtures not work in conftest.py?

### DISCUSSION THREAD

**Discussion by jaraco** [pytest-dev/pytest#12677] [Q&A] [answered]
I'd like to enable a third-party fixture for all tests in the test suite.

I thought I could do this by [adding a `pytestmark = pytest.mark.usefixtures(fixturename)` to conftest.py](https://github.com/jaraco/jaraco.vcs/commit/49c9d80da5399acbf16b4e3b49d5a96ea6cbfb26), but it turns out that fails to trigger the fixture. I ended up [creating an autouse fixture that depends on the desired fixture](https://github.com/jaraco/jaraco.vcs/commit/689c2a7d0a47da22efb0b43aafb793342a734107) to make it work, but that feels heavy-handed and creates an extra fixture name that I'd rather not have.

Is there a way to declare to use a fixture to be used for all tests? I [asked Gemini](https://g.co/gemini/share/00a603820477), but it offers wrong answers only.

Pytest version is latest (8.3.2).

**Comment by RonnyPfannschmidt** [comment]
conftests are plugins, not part of the node tree, thats why a marker on them will not propagate (for good reason)

the recommended way to have a fixture happen for all tests is a autouse fixture

**Reply by jaraco** [reply]
So there is no way to declare to "auto-use a third party fixture" without wrapping it in another fixture?

**Reply by RonnyPfannschmidt** [reply, marked as answer]
The alternative would be to add the marker to the session, which is hacky in another way 

The recommended Way to activate a fixture for everything is to have a autouse fixture depend on it
